package.json

```diff
@@ -102,16 +102,29 @@
     "extends": [
       "config:base"
     ],
-    "ignoreDeps": [
-      "prettier"
-    ],
     "lockFileMaintenance": {
       "enabled": true
     },
-    "pinVersions": false,
-    "prCreation": "not-pending",
+    "ignoreDeps": [
+      "prettier"
```

Comment:
Huh, why are you ignoring prettier updates?

Reply:
I always did, to be honest, as prettier has a rather interesting versioning strategy..:

> We recommend pinning an exact version of prettier in your package.json as we introduce stylistic changes in patch releases.

This is taken straight from the prettier [docs](https://prettier.io/docs/en/install.html).